### PR TITLE
Issue #29 - Remove customer Specific info from .phtml files

### PR DESCRIPTION
**Pull request by srenon**

### CODE REVIEW

File: view/frontend/web/js/customer.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * https://www.magepal.com | support@magepal.com
+ */
+
+define([
+    'underscore',
+    'Magento_Customer/js/customer-data'
+], function (_, customerData) {
+    'use strict';
+    
+    window._learnq = window._learnq || [];
```
Comment (remstone7) [inline view/frontend/web/js/customer.js]: do we need the `.window` here or just use `var` ?

File: Plugin/Customer/Model/CustomerData.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    /**
+     * @param Customer $subject
+     * @param $result
+     * @return mixed
+     */
+    public function afterGetSectionData(Customer $subject, $result)
+    {
+        if ($this->currentCustomer->getCustomerId()) {
+            $result['lastname'] = $this->currentCustomer->getLastname();
```
Comment (remstone7) [inline Plugin/Customer/Model/CustomerData.php]: can we add `firstname`?

Comment (srenon) [reply Plugin/Customer/Model/CustomerData.php]: The first name is added by default magneto

see Magento\Customer\CustomerData\Customer

    public function getSectionData()
    {
        if (!$this->currentCustomer->getCustomerId()) {
            return [];
        }

        $customer = $this->currentCustomer->getCustomer();
        return [
            'fullname' => $this->customerViewHelper->getCustomerName($customer),
            'firstname' => $customer->getFirstname(),
            'websiteId' => $customer->getWebsiteId(),
        ];

File: view/frontend/web/js/customer.js (hunk 1 above)
Comment (srenon) [reply view/frontend/web/js/customer.js]: In a function, the variable will be a local variable to that function so you need "window." to get access to the global namespace of `_learnq`

**Pull request merged**